code/modules/mob/mob_movement.dm

```diff
@@ -175,6 +175,9 @@
 	if(mob && .)
 		mob.throwing = 0
 
+	for(var/obj/O in mob)
+		O.on_mob_move(direct, src)
```

Comment:
I mean, it's either make a proc that everything after this PR if it is merged can use, or check specifically for a flightpack?